Source/WebKit/UIProcess/GPU/GPUProcessProxy.cpp

```diff
@@ -475,8 +476,13 @@ void GPUProcessProxy::didFinishLaunching(ProcessLauncher* launcher, IPC::Connect
 #endif
 
 #if PLATFORM(COCOA)
-    if (auto networkProcess = NetworkProcessProxy::defaultNetworkProcess())
+    if (auto networkProcess = NetworkProcessProxy::defaultNetworkProcess()) {
         networkProcess->sendXPCEndpointToProcess(*this);
+        String uiProcessName;
+        auto displayName = makeString(WEB_UI_STRING("%s Graphics and Media", "visible name of the GPU process. The argument is the application name."), uiProcessName);
```

Comment:
The `makeString` function won’t be able to substitute a string for `%s`. That can be done a variety of other ways, but this code won’t work as is.

Reply:
Ah, yes, that is a good catch.

Thanks for reviewing!